### PR TITLE
Remove live sample link example

### LINKED ISSUE #34326: Remove 2 links to yari-demos.prod.mdn.mozit.cloud
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/API/Window/resize_event#window_size_logger

### What specific section or headline is this issue about?

Window size logger

### What information was incorrect, unhelpful, or incomplete?

There are two links to `yari-demos.prod.mdn.mozit.cloud`, which is now decommissioned: https://github.com/search?q=org%3Amdn+yari-demos+repo%3Amdn%2Fcontent&type=code

The other occurence is in the Writing guidelines and suggests to use these links:  https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Page_structures/Live_samples#grouping_code_blocks

### What did you expect to see?

The links may be replaced by something like this: https://live.mdnplay.dev/en-US/docs/MDN/Writing_guidelines/Page_structures/Live_samples/_sample_.grouping_code_blocks_by_heading.html?id=grouping_code_blocks_by_heading&amp;url=%2Fen-US%2Fdocs%2FMDN%2FWriting_guidelines%2FPage_structures%2FLive_samples

### Do you have any supporting links, references, or citations?

_No response_

### Do you have anything more you want to share?

_No response_

### PR DESCRIPTION
**Pull request by Josh-Cena**
Fix https://github.com/mdn/content/issues/34326. This should not be explained anyway.

### CODE REVIEW

File: files/en-us/mdn/writing_guidelines/page_structures/live_samples/index.md [hunk 1]
```diff
@@ -109,7 +109,7 @@ Only the `<p>` element with `class="fancy"` will get styled `red`:
 \{{EmbedLiveSample("paragraph-styling")}}
 ````
 
-The macro uses a special URL to fetch the output for a given group of code blocks, for example `https://yari-demos.prod.mdn.mozit.cloud/en-US/docs/Web/CSS/animation/_sample_.Cylon_Eye.html`. The general structure followed is `https://url-of-page/_sample_.group-id.html`, where `group-id` is the ID of the heading or block where the code blocks are located.
+The macro uses a special URL to fetch the output for a given group of code blocks. The ID will be a part of the URL, but otherwise this URL should be treated as opaque and should not be hardcoded in content. If you need to refer to it, use the `LiveSampleLink` macro.
```
Comment (hamishwillee) [inline files/en-us/mdn/writing_guidelines/page_structures/live_samples/index.md:112]: What do you mean "if you need to refer to it?". As I understand it the `LiveSampleLink` macro generates a link that will open an example in a new page - so you might be saying "if you need to open the example in a new page you can use the id in the `LiveSampleLink` macro." ?

Comment (Josh-Cena) [reply files/en-us/mdn/writing_guidelines/page_structures/live_samples/index.md:112]: Yes, I think that's what I meant, that you should never create a Markdown link that hardcodes the URL. Rephrasing welcome.

Comment (hamishwillee) [reply files/en-us/mdn/writing_guidelines/page_structures/live_samples/index.md:112]: Borrowing your words ...

```suggestion
The macro uses a special URL that includes the ID to fetch the output for a given group of code blocks. You should never use this URL directly — if you need to link to the example use the [LiveSampleLink](#livesamplelink_macro) macro.
```

Comment (Josh-Cena) [reply files/en-us/mdn/writing_guidelines/page_structures/live_samples/index.md:112]:
```suggestion
The macro uses a special URL that includes the ID to fetch the output for a given group of code blocks. You should never hardcode this URL directly in content — if you need to link to the example, use the [`LiveSampleLink`](#livesamplelink_macro) macro.
```

**Review by hamishwillee — APPROVED** [review]
Thanks very much.

**Pull request merged**